docker/entrypoint.sh

```diff
@@ -1,15 +1,11 @@
 #!/bin/bash
 set -e
 
-# If not provided set to public facing IP address of this node
-EXTERNAL_PEER_ADDRESS="${EXTERNAL_PEER_ADDRESS:-http://$(curl -s https://api.ipify.org):3013/}"
```

Comment:
I see not needed any longer as peer receiving ping will use IP from inbound TCP connection